cmiyc/vanilla_vae.py

```diff
@@ -63,11 +68,14 @@ def vae_loss(inputs, outputs, original_dim, z_mean, z_log_var):
         kl_loss = -0.5 * K.sum(kl_loss, axis=-1)
         return K.mean(reconstruction_loss + kl_loss)
 
-    def get_data(self, sig_id=1, sig_type='genuine'):
+    def load_data(self, sig_id=1, sig_type='genuine'):
         '''
         Load the specified sig id and signature type.
+        
+        TODO: again, wasteful to be calling this here.
```

Comment:
Maybe we could decouple the `VanillaVae` and the `load_data` functionality simply by having VanillaVae's train function accept `x_train` and `y_train` as parameters? That is nothing critical tho

Reply:
^ agreed, ideally we should have something that wraps around the VanillaVAE training and the experiment. But as it currently stands, since we save the training weights and load them into the experiment later, this works, even if it's inefficient. Totally fine w you refactoring this later (or I can do it if there isn't something more urgent)